vizro-core/src/vizro/models/_components/form/_form_utils.py

```diff
@@ -54,7 +54,7 @@ def validate_value(cls, value, values):
         [entry["value"] for entry in values["options"]] if isinstance(values["options"][0], dict) else values["options"]
     )
 
-    if value and not is_value_contained(value, possible_values):
+    if value and ALL_OPTION not in value and not is_value_contained(value, possible_values):
```

Comment:
The `possible_values` here is always a `self.options` prop that does not contain "ALL", and this validation function is invoked from the `_build_dynamic_placeholder` method. Previously `self.value` was not set anywhere from code, so this wan't failed.